wrappers/java/wrapper

```diff
@@ -33,23 +33,27 @@ wrapper_dir=$(dirname "$0")
 cd "$wrapper_dir"
 
 if [ "$action" == "version" ]; then
-	exec ant -S -q version
+	exec mvn -S -q version
```

Comment:
This fails because the `-S` option doesn't exist.
I thought that the test-suite itself would fail because of that. I need to have a look.
This is supposed to be printing the version of the wrapper (currently "3.1")

Reply:
This mode is not essential. It is only there for convenience so that one can run `./wrapper version` to get the version.

I have found a solution on https://stackoverflow.com/questions/3545292/how-to-get-maven-project-version-to-the-bash-command-line
```mvn -q -Dexec.executable=echo -Dexec.args='${project.version}' --non-recursive exec:exec```